wallets/cosmostationmobile.json

```diff
@@ -0,0 +1,73 @@
+{
+  "$schema": "../wallet.schema.json",
+  "wallet_name": "cosmostation",
+  "pretty_name": "cosmostation",
+  "website": "https://www.cosmostation.io/",
+  "git_repo": "https://github.com/cosmostation/cosmostation-android",
+  "supported_chains": [
+    "cosmoshub",
+    "akash",
+    "assetmantle",
+    "axelar",
+    "bandchain",
+    "bitcanna",
+    "bitsong",
+    "cerberus",
+    "chihauha",
+    "comdex",
+    "crescent",
+    "cryptoorgchain",
+    "cudos",
+    "desmos",
+    "emoney",
+    "evmos",
+    "fetchhub",
+    "gravitybridge",
+    "injective",
+    "iris",
+    "impacthub",
+    "juno",
+    "kava",
+    "ki",
+    "konstallation",
+    "kujira",
+    "likecoin",
+    "lumnetwork",
+    "panacea",
+    "nyx",
+    "omniflix",
+    "onomy",
+    "osmosis",
+    "passage",
+    "persistence",
+    "provenance",
+    "regen",
+    "rizon",
+    "secretnetwork",
+    "sentinel",
+    "shentu",
+    "sifchain",
+    "sommelier",
+    "stargaze",
+    "starname",
+    "stride",
+    "teritori",
+    "tgrade",
+    "umee",
+    "xpla"
+  ],
+  "platforms": [
+    {
+      "device": "mobile",
+      "type": "application",
+      "platform": "ios",
+      "install_link": "https://apps.apple.com/kr/app/cosmostation/id1459830339",
+    },
+    {
+      "device": "mobile",
+      "type": "application",
+      "platform": "android",
+      "install_link": "https://play.google.com/store/apps/details?id=wannabit.io.cosmostaion",
```

Comment:
Seems legit, even though the misspelling of "cosmostation" make it looks suspicious.